Makefile

```
@@ -132,14 +132,35 @@ preflight:
+	fi
+	@bash tool/create_worktree.sh "$(name)" "$(dir)" "$(task)"
+
+migrate-test: start-firebase-emulators
+	@echo "Running local database migrations..."
+	@dart run tool/deploy/deploy_schema.dart
+
 # --- Automation & Gemini Targets ---
 
-.PHONY: data feature research resolve-comments unit-tests integration-tests analyze-architecture
+.PHONY: data seed-local seed-remote feature research resolve-comments unit-tests integration-tests analyze-architecture
 
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)